src/Schema/Builder.php

```diff
@@ -165,9 +171,23 @@ public function getTables()
         return $collections;
     }
 
-    public function getTableListing()
+    public function getTableListing($schema = null, $schemaQualified = true)
```

Comment:
Would this function benefit from a doc block to document `$schema` as `string|array|null`, and `$schemaQualified` as a `bool`?